app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -0,0 +1,51 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/train-to-be-a-teacher/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
+
+            <p>
+                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options depending on your situation.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-have-experience" class="category__nav-card">
+            <h2>If you have classroom experience</h2>
+
+            <p>
+                If you have already spent some time working in the classroom, you may be able to follow the assessment only route to show you meet QTS.
+            </p>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-want-to-change-career" class="category__nav-card">
+            <h2>If you want to change career</h2>
+
+            <p>
+            Use the experience you’ve gathered so far to inspire students and bring your skills and expertise to the classroom.
```

Comment:
I like this snippet but wonder if we need to link it more to the training content underneath rather than the benefits of career changers as teachers?